src/main/webapp/app/entities/quiz-exercise/quiz-exercise-detail.component.ts

```diff
@@ -567,6 +615,22 @@ export class QuizExerciseDetailComponent implements OnInit, OnChanges, OnDestroy
         return reasonString.substr(0, reasonString.length - 5);
     }
 
+    /**
+     * @function warningReasonsHTML
+     * @desc Get the reasons, why the quiz is invalid as an HTML string
+     * @return {string} the reasons in HTML
+     */
+    warningReasonsHTML(): string {
```

Comment:
This function has exactly the same code as the function invalidReasonsHTML() above, which is duplicated code and quite bad. Please refactor this into one function call in which you e.g. pass the list of invalidReasons or warningReasons

Reply:
Made one function out of it